src/Controller/FrameworksController.php

```diff
@@ -297,6 +297,10 @@ public function listByPillar(Request $request, string $pillar, int $page = 1)
      */
     public function search(Request $request, int $page = 1)
     {
+        // Get feature flag if it exists &feature=Scale
+        $flag = filter_var($request->query->get('feature'), FILTER_SANITIZE_STRING);
```

Comment:
Good comment, but it's not what we are doing in the code.

Reply:
yes it is, that code gets it from the query and the structure it is done in the query is &feature=guidedmatch. I will change the 'scale' to 'guidedmatch'.